pkg/logql/rangemapper_test.go

```diff
@@ -1675,7 +1753,7 @@ func Test_SplitRangeVectorMapping(t *testing.T) {
 		// Non-splittable vector aggregators - should go deeper in the AST
 		{
 			`topk(2, count_over_time({app="foo"}[3m]))`,
-			`topk(2, 
+			`topk(2,
```

Comment:
I'm seeing a few of these changes in this PR where trailing spaces are removed.
I'd suggest you commit with `-p` to only include the changes relevant to the PR; it makes the PR larger and adds some irrelevant detail.